index.js

```diff
@@ -136,6 +136,19 @@ const hourPoller = cron.job('0 0 * * * *', async function() {
         console.log(
             'ERROR: Could not find PUG poll channel when creating new poll.');
       }
+
```

Comment:
I don't think we need this. We already clear the announce channel above when we delete the previous poll (i.e. Monday at midnight, which is when the new poll actually starts taking effect).

Reply:
I think this was pulled in by a merge. It might have been something that used to exist and was removed?

Reply:
Done.